tests/cypress/e2e/unit_tests/p0_fleet.spec.ts

```diff
@@ -393,3 +393,41 @@ if (!/\/2\.7/.test(Cypress.env('rancher_version')) && !/\/2\.8/.test(Cypress.env
     );
   })
 };
+
+if (!/\/2\.8/.test(Cypress.env('rancher_version')) && !/\/2\.9/.test(Cypress.env('rancher_version'))) {
+  describe('Test GitJob security context',  { tags: '@p0' }, () => {
+    qase(160,
+      it('FLEET-160: Test GitJob pod security context', { tags: '@fleet-160' }, () => {
+        // Check the GitJob pod for Security Context.
+        cy.accesMenuSelection('local', 'Workloads', 'Pods');
+        cy.filterInSearchBox('gitjob');
+        cy.verifyTableRow(0, 'Running', 'gitjob');
+        cy.contains('gitjob').click();
+        cy.clickButton('Config');
+        cy.get('section#container-0')
+          .find('.side-tabs ul.tabs li')
+          .eq(3)
+          .should('have.id', 'securityContext')
+          .should("be.visible")
```

Comment:
Q: does the double `should` work? If you try the negative version correctly fails?

Reply:
Yes, double `should` work, I saw here -> https://docs.cypress.io/api/commands/should#Yields

Reply:
What I mean is that if the negative works as well? What I want to make sure is that it does not fail silently.
Sometimes I have seen how if the locator is not valid a double `should` will not detect it.
This why I ask if the negative version `should.not` guarantees it fails or if a bad locator guarantees the assertion will cath it properly?

Reply:
| Bad Locator | Good Locator |
|--------|--------|
| ![image_480](https://github.com/user-attachments/assets/0a706ee2-8a63-4573-9dc3-647434829a68) | ![image_480](https://github.com/user-attachments/assets/12958ff0-f1d2-4ad0-a4b0-e394c133604b) |